### DISCUSSION TITLE
Any place that describes what's planned for v4?

### DISCUSSION THREAD

**Discussion by joshpetit** [isar/isar#1303] [Help]
I hear mention of features being planned for v4, but I don't see any issue or project that tracks what's planned for that release. Is there a roadmap so we can know what to expect in the future?

**Comment by simc** [comment]
Good idea let me create a road map issue 🙂

**Reply by joshpetit** [reply]
I completely understand if a roadmap is something you don't have bandwidth to create, I just want to ping here in case it slipped from your todo list.
